runtime/ieee_754.js

```diff
@@ -200,7 +200,7 @@ function caml_classify_float(x) {
     if (x != 0) return 1;
     return 2;
   }
-  return isNaN(x) ? 4 : 3;
+  return Number.isNaN(x) ? 4 : 3;
```

Comment:
FYI, the two are not strictly equivalent.
> Number.isNaN() doesn't attempt to convert the parameter to a number, so non-numbers always return false

Reply:
Yeah, but I don't think it's a problem here. Am I missing something?

Reply:
It is correct here, but it's easy to miss the difference in behavior when mixed in the middle of other changes. It would simplify review if the PR was mentioning such difference.